scripts/superdesk/notification/notification.js

```diff
@@ -71,10 +71,15 @@
             ws.onopen = function(event) {
                 $interval.cancel(connectTimer);
                 $rootScope.$broadcast('connected');
+                notified = null;
             };
 
             ws.onclose = function(event) {
-                $rootScope.$broadcast('disconnected');
+                if (!notified) {
+                    $rootScope.$broadcast('disconnected');
+                    notified = true;
```

Comment:
sorry missed that, working on it, so if its closed by click, then notification would re-appear on next timeout, if disconnection persist.